cms/djangoapps/contentstore/views/item.py

```diff
@@ -696,19 +700,17 @@ def create_item(request):
     """
     Exposes internal helper method without breaking existing bindings/dependencies
     """
-    return _create_item(request)
+    return _create_item(request.user, request.json)
```

Comment:
Should this perhaps be:
```suggestion
    return _create_item(request.user, request.data)
```
Just suggesting because you've consistently changed to request.data in other places. I'm not aware of the different here though.

Reply:
The existing flow takes one standard `WSGIRequest` objects and is then enriched with the `expect_json` decorator to have the input data set on `request.json`.

When using django-rest-framework, the request object is `rest_framework.request.Request`, which provides the attribute request.data with the parsed input - i.e, it is basically doing the same thing as the our function is doing except for the authentication handling/media negotiation/etc.

The change from request.json to request.data is needed to reflect this.